packages/vscode-extension/src/webview/components/NetworkBar.tsx

```diff
@@ -0,0 +1,51 @@
+import IconButton from "./shared/IconButton";
+import { useNetwork } from "../providers/NetworkProvider";
+
+const NetworkBar = () => {
```

Comment:
I think we typically use `function NetworkBar` type of syntax for components. Would prefer to have this consistent across the repo